css/style.css

```diff
@@ -255,42 +262,109 @@ button {
 
 /**************** Hero section **************************/
 
-.hero-box {
-  padding-top: 141px;
-  background-image: url("../img/bg-s/star-big.svg"),
-    linear-gradient(to top, rgb(4, 11, 17), rgba(4, 11, 17, 0)),
+.hero-section {
+  background-image: linear-gradient(
+      to top,
+      rgb(4, 11, 17) 15%,
+      rgba(4, 11, 17, 0) 55%
+    ),
     radial-gradient(
-      ellipse at center,
-      rgba(48, 113, 101, 0.3) 10%,
-      rgba(20, 27, 34, 0.3) 70%
+      ellipse at top left,
+      rgba(48, 113, 101, 0.8) 10%,
+      rgba(20, 27, 34, 0.6) 70%
     );
-  background-size: 496px auto, cover, 950px 278px;
-  background-position: bottom -50px right -170px, bottom, top -100px left -200px;
   background-repeat: no-repeat;
+  background-size: auto, 750px 278px;
+  background-position: bottom, top -100px left 0px;
   background-color: var(--secondary-color);
+  position: relative;
+  overflow: hidden;
+}
+
+.hero-section::before {
+  content: " ";
+  width: 496px;
+  height: 496px;
+  display: inline-block;
+  background-image: url("../img/bg-s/star-big.svg");
+  background-size: 496px;
+  background-repeat: no-repeat;
+  transform: rotate(-30deg);
+  position: absolute;
+  right: -150px;
+  bottom: -60px;
+  opacity: 70%;
+}
+
+.hero-box {
+  display: flex;
+  align-items: center;
+}
+
+.hero-image {
+  flex-shrink: 0;
+  padding-left: 150px;
+  position: relative;
 }
 
 .hero-content {
-  width: 750px;
+  width: 600px;
+  padding-top: 140px;
+  padding-bottom: 190px;
 }
 
 .hero-title {
+  width: inherit;
   font-size: 72px;
   line-height: 1.11;
   text-transform: uppercase;
 }
 
+.last-line::before {
+  content: "";
+  white-space: pre;
+  display: block;
+}
+
 .hero-text {
   font-family: var(--text-font);
   font-size: 20px;
   line-height: 1.5;
-  margin: 20px 0 24px;
+  padding: 20px 0 24px;
 }
 
 .hero-text::first-letter {
   text-transform: capitalize;
 }
 
+.hero-counter {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: 1fr 0.4fr;
+  grid-template-areas:
```

Comment:
Nice work with grids! 
If you would like do it more adaptively, then try to do it so that when you add more properties and values, nothing breaks and they appear